assets/src/edit-story/constants.js

```diff
@@ -45,10 +45,6 @@ export const DEFAULT_DPR = 0.5;
 // Default 1em value for font size.
 export const DEFAULT_EM = PAGE_HEIGHT * 0.02186;
 
-// @todo Confirm real min-max font sizes.
-export const MIN_FONT_SIZE = 30;
-export const MAX_FONT_SIZE = 200;
-
```

Comment:
What's with this change here?

Reply:
Super unrelated change but actually dead code